packages/widgetbook_core/lib/src/navigation_tree/models/leaf_navigation_node_data.dart

```diff
@@ -1,9 +1,20 @@
 import 'package:widgetbook_core/widgetbook_core.dart';
 
-class LeafNavigationNodeData extends NavigationTreeNodeData {
-  LeafNavigationNodeData({
+class LeafNavigationNodeData extends NavigationNodeDataInterface {
```

Comment:
Since `NavigationNodeDataInterface` is an abstract (non-freezed) class, I think we can make `LeafNavigationNodeData` a freezed class.

Reply:
Couldn't do this since `WidgetbookUseCase` extends this class